htdocs/install/mysql/tables/llx_establishment.sql

```diff
@@ -0,0 +1,41 @@
+-- ============================================================================
+-- Copyright (C) 2015       Alexandre Spangaro  <aspangaro.dolibarr@gmail.com>
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <http://www.gnu.org/licenses/>.
+--
+-- ============================================================================
+--
+-- Structure de la table llx_establishment
+--
+
+CREATE TABLE IF NOT EXISTS llx_establishment (
+  rowid 			integer NOT NULL auto_increment PRIMARY KEY,
+  entity 			integer NOT NULL DEFAULT 1,
+  name				varchar(50),
+  address           varchar(255),
+  zip               varchar(25),
+  town              varchar(50),
+  fk_state          integer DEFAULT 0,
+  fk_country        integer DEFAULT 0,
+  profid1			varchar(20),
+  profid2			varchar(20),
+  profid3			varchar(20),
+  phone				varchar(20),
+  fk_user_author 	integer NOT NULL,
+  fk_user_mod		integer NOT NULL,
+  datec				datetime NOT NULL,
+  tms				timestamp NOT NULL,
+  statut            tinyint DEFAULT 1,
```

Comment:
Since this is a new module, I think the correct english "status" should be used instead.